t/Renard/Curie/Component/ScrollZoom.t

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env perl
+
+use Test::Most;
+
+use lib 't/lib';
+use Renard::Curie::Setup;
+use Renard::Curie::App;
+use Renard::Curie::Model::Document::PDF;
+use CurieTestHelper;
+use Renard::Curie::Types qw(Int InstanceOf);
+use Function::Parameters;
+no strict;
+
+my $cairo_doc = CurieTestHelper->create_cairo_document;
+
+fun Scroll_Event( (InstanceOf['Renard::Curie::App']) $app) {
+	my $event = Gtk3::Gdk::Event->new('scroll');
+	$event->state( 'control-mask' );
+	$event->direction('down');
+	$app->page_document_component->signal_emit( 'scroll_event' => $event );
+}
+
+subtest 'Check that ctrl+scroll-up zooms into the page' => fun {
+	my ( $app, $page_comp ) = CurieTestHelper->create_app_with_document($cairo_doc);
+
+	print STDERR $page_comp->zoom_level . "\n";
+
+	ok($page_comp->zoom_level - 1.0 < .001, 'Start at Zoom Level 1.0' );
```

Comment:
Nice catch on using floating point tolerance to compare the numbers here. I'm not sure it is necessary since any number 53-bit integer should be representable exactly. But if we do need FP equality testing, I suggest we use [`Test::Number::Delta`](https://p3rl.org/Test::Number::Delta).